jsk_aibo_robot/aibo_selenium_ros/python/aibo_selenium_ros/geckodriver.log

```diff
@@ -0,0 +1,71 @@
+1694083040310	geckodriver	INFO	Listening on 127.0.0.1:36887
```

Comment:
In order to keep the code base minimal, files that are generated later, such as log files, generally speaking, should not be included in the git repository.

Reply:
@a-ichikura Could you update `jsk_aibo_robot/aibo_selenium_ros/.gitignore` so that this file will not be added to git repo?